plutus-tx-plugin/test/TH/all.plc.golden

```diff
@@ -1,7 +1,7 @@
 (abs
-  out_Bool_173
+  out_Bool_186
```

Comment:
It always looks weird to me when a simplification results in uniques having greater indices.

Reply:
Yeah, I know. But we do so much renaming it's hard to know where things are coming from these days. Maybe the real issue is that I still have tests that aren't using de bruijn indices :thinking:

Reply:
> Maybe the real issue is that I still have tests that aren't using de bruijn indices thinking

No, I think in this case syntactic equality serves us well. I mean, why do the uniques have greater indices? When you simplify things, uniques normally become smaller. Uniques being bigger is usually a sign of things becoming more complex and inefficient. Not that this is a valid well-defined metric, just wondering.

Reply:
Well, we will use slightly more nodes for a *single* recursive function now (which this is) since we will let-bind `fix`. I guess that's what this is.

Reply:
Ah, I see.